<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Abe-laptop" ignored="0" name="Command line test" tests="7" failures="1" timestamp="2022-05-22T15:55:09 CDT" time="26.346" errors="0">
  <testcase name="testAppleTitle" time="1.876" classname="scripts._01_AppleTest"/>
  <testcase name="testAppleURL" time="0.829" classname="scripts._01_AppleTest"/>
  <testcase name="X" time="0.0" classname="scripts._01_AppleTest">
    <failure type="java.lang.AssertionError" message="Failure on purpose">
      <![CDATA[java.lang.AssertionError: Failure on purpose
at scripts._01_AppleTest.X(_01_AppleTest.java:51)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- X -->
  <testcase name="testApplicationFormHeading" time="0.652" classname="scripts._02_TechGlobalTest"/>
  <testcase name="testEtsyLogo" time="1.893" classname="scripts._03_EtsyTest"/>
  <testcase name="testEtsySearch" time="1.981" classname="scripts._03_EtsyTest"/>
  <testcase name="testEtsySearchResult" time="3.998" classname="scripts._03_EtsyTest"/>
</testsuite> <!-- Command line test -->
